src/RobotFrameworkAI/modules/Module.py

```diff
@@ -101,17 +105,21 @@ def get_default_values_for_common_arguments(
     def validate_common_input_arguments(self, max_tokens:int, temperature:float, top_p:float, frequency_penalty:float, presence_penalty:float):
         error_messages = []
         if not self.is_valid_max_tokens(max_tokens):
-            error_messages.append(f"Invalid value '{max_tokens}' for 'max_tokens'. Value must be greater than 0 and less than or equal to 4096.")
+            error_messages.append(f"Invalid value `{max_tokens}` for `max_tokens`. Value must be greater than 0 and less than or equal to 4096.")
```

Comment:
4096 - this is true only for certain models. So if we want to write the actual number, we should have a dictionary/list somewhere with the model = max_tokens